addon/components/o-s-s/text-area.hbs

```diff
@@ -0,0 +1,9 @@
+<div class="fx-1">
+  <div class="oss-textarea-container {{if @errorMessage ' oss-textarea-container--errored'}}">
+    <Textarea {{on "change" (fn this._onChange @value)}} @value={{@value}} placeholder={{@placeholder}}
+              disabled={{@disabled}} class="oss-textarea {{this.computedClass}}" rows={{this.rows}} ...attributes />
```

Comment:
having the `...attributes` here might cause some issues when targeting the component imo